agasc/supplement/magnitudes/update_mag_supplement.py

```diff
@@ -277,18 +269,7 @@ def update_supplement(agasc_stats, filename, include_all=True):
 
                 new_stars = outliers_new[new_stars]['agasc_id']
 
-    if outliers is None:
-        logger.warning('Creating new "mags" table')
-        outliers = outliers_new
-        new_stars = outliers_new['agasc_id']
-        updated_stars = np.array([], dtype=mags_dtype)
-
-    mode = 'r+' if filename.exists() else 'w'
-    with tables.File(filename, mode) as h5:
-        if 'mags' in h5.root:
-            h5.remove_node('/mags')
-        h5.create_table('/', 'mags', outliers)
-    save_version(filename, 'mags')
+    update_mags_table(outliers)
```

Comment:
This looks not to match the current function definition for update_mags_table?

Reply:
That is correct. I am actually undoing this change, because one does not gain much by calling the update function. The update itself is different.